packages/devtools_app/lib/src/screens/inspector_v2/inspector_screen.dart

```diff
@@ -63,9 +62,6 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
   InspectorTreeController get _summaryTreeController =>
```

Comment:
we could probably also rename every thing from "summaryTree" to "widgetTree" or "inspectorTree"

Reply:
Good point! Caught some more large code deletions that way